starcli/search.py

```diff
@@ -4,6 +4,7 @@
 from datetime import datetime, timedelta
 from time import sleep
 import logging
+from rich.logging import RichHandler
```

Comment:
this should be on line 17 with the other third-party imports

Reply:
Okay. Committing the changes in a while.